.gitignore

```diff
@@ -20,3 +20,6 @@ atlassian-ide-plugin.xml
 node
 node_modules
 core.wcm.components.commons.datalayer.*
+*/**/META-INF/MANIFEST.MF
+*/**/package-lock.json
+jacoco.exec
```

Comment:
These should not be necessary

Reply:
Please let me know if you want me to undo the changes? Thank you !

Reply:
Taken care of !